README.md

```diff
@@ -244,6 +244,15 @@ Default options act as default for all components
 - fileformat
   - icons_enabled (true)\
    Whether to displays icon before component
+- signify
+  - colored        (true)\
+    Whether to show colors
```

Comment:
oh and readme should imply that when this option is set the colors are automatically set from current vim colorscheme. Same change to docs

Reply:
Done